builder/role_image.go

```diff
@@ -168,9 +171,14 @@ func (r *RoleImageBuilder) CreateDockerfileDir(role *model.Role, baseImageName s
 
 		// Symlink spec configuration file
 		// from <ROOT_DIR>/opt/hcf/specs/<ROLE_NAME>/<JOB>.json
-		specConfigSource := filepath.Join(jsonSpecsDir, role.Name, job.Name+configstore.JobConfigFileExtension)
+		specConfigSource := filepath.Join(jsonSpecsDir, role.Name, job.Name+model.JobConfigFileExtension)
+		err = job.WriteConfigs(role, specConfigSource, r.lightOpinionsPath, r.darkOpinionsPath)
```

Comment:
There's no point in symlinks anymore, since you can just create the file at the destination directly instead (they no longer in the packages layer).